content/docs/components/datepicker.mdx

```diff
@@ -69,6 +69,12 @@ Use the `inline` prop to show the datepicker component without having to click i
 
 <Example name="datepicker.inline" />
 
+## Date value
```

Comment:
[Controlled](https://react.dev/reference/react-dom/components/input#controlling-an-input-with-a-state-variable) Date/Datepicker is more appropriate here.